plugins/azure/test/compute_test.py

```diff
@@ -75,7 +75,15 @@ def test_ssh_key(builder: GraphBuilder) -> None:
 
 def test_virtual_machine(builder: GraphBuilder) -> None:
     collected = roundtrip_check(AzureVirtualMachine, builder)
-    assert len(collected) == 4
+    assert len(collected) == 2
+
+
+def test_virtual_machine_resources(builder: GraphBuilder) -> None:
+    collected = roundtrip_check(AzureVirtualMachine, builder)[0]
+    assert collected.instance_cores == 0.0
+    assert collected.instance_memory == 0.0
+    assert collected.instance_type == "Standard_A0"
+    assert collected.instance_status == InstanceStatus.RUNNING
```

Comment:
Do i need to create test that assert more collected resources?

Reply:
not for virtual machine, but disk, snapshot, vmSize, etc.

Reply:
can't add AzureSnapshot resource test, because it collect an empty list:

```
---------------------------------------------------- Captured stdout call -----------------------------------------------------
[]
```

Reply:
You can. The test setup uses static files to capture an example result.
For snapshots it is [this file](https://github.com/someengineering/resoto/blob/main/plugins/azure/test/files/compute/snapshots.json), which is empty.

You can copy & paste an example from the [Azure REST docs for snapshots](https://learn.microsoft.com/en-us/rest/api/compute/snapshots/list?tabs=HTTP): copy the sample response.

You should see two snapshots.